### PR TITLE
ci/e2e: add a ssh connection test

### PR DESCRIPTION
**Pull request by ldevulder**

### CODE REVIEW

File: tests/e2e/bootstrap_with_rancher_test.go [hunk 1]
```diff
@@ -320,5 +333,22 @@ var _ = Describe("E2E - Bootstrapping node with Rancher", Label("bootstrapping")
 			// Check that the VM is added
 			Expect(internalClusterName).To(Equal(internalClusterToken))
 		})
+
+		By("Checking VM ssh connection", func() {
+			// TODO: Create a native Go function for this
+			ip, err := exec.Command("sed", "-zn", "/name='"+vmName+"'/s/.*ip='\\(.*\\)'.*/\\1/p", "../assets/net-default.xml").CombinedOutput()
```
Comment (mudler) [inline tests/e2e/bootstrap_with_rancher_test.go:339]: is this a leftover or I am missing something here? I see it is doing a sed here, but nothing consuming the changed file

Comment (ldevulder) [reply tests/e2e/bootstrap_with_rancher_test.go:339]: The file is not changed, this line is to get the IP address of the node from the `default` network, as there is no DNS. The `ip` var is then used in the ssh command to run a `uname -n`.

Comment (mudler) [reply tests/e2e/bootstrap_with_rancher_test.go:339]: oh right, I was fooled by the sed command here :)

**Review by mudler — APPROVED** [review]

**Comment by ldevulder** [comment]
Verification run on my lab: [ssh_verification_run.log](https://github.com/rancher-sandbox/os2/files/8707020/ssh_verification_run.log)

**Pull request merged**